.circleci/config.yml

```diff
@@ -19,7 +19,7 @@ commands:
             - v6-yarn-deps-
       - run:
           name: Install dependencies with Yarn
-          command: yarn install --frozen-lockfile
+          command: yarn install --frozen-lockfile --ignore-platform
```

Comment:
This is done to resolve circleci pipeline issue - https://app.circleci.com/pipelines/github/okta/okta-developer-docs/5450/workflows/215430a8-2447-40e6-9147-0770001c9c5e/jobs/12442

Reply:
ok, lets proceed with this...